.github/actions/tune-runner-vm/action.yml

```diff
@@ -0,0 +1,13 @@
+name: Tune Runner VM
+description: tunes the GitHub Runner VM operation system
+runs:
+  using: composite
+  steps:
+    - run: |
+        if [[ "$OSTYPE" == "linux-gnu"* ]]; then
```

Comment:
@lhotari, is it a permanent issue or temporary? (I assume so?) If it's temporary, I think we already have a minimized decent workaround. I would just leave it as is for now.